### PR TITLE
refactor(sdk): update available sdk clients

### PR DESCRIPTION
**Pull request by kneal**
This PR will simplify the available documentation on the documentation site. The documentation site is not the source of truth for this documentation and tends to be out of date. This new style will provide where to get the docs along with a simple example.

Content changes:

* Removed Go sdk docs in favor of Godoc
* Adding Go sdk Splash page
* Added Python sdk splash page

### CODE REVIEW

**Review by matt-fevold — DISMISSED** [review]

File: content/reference/sdk/go.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Get build info example
+
+Below is a sample Go program demonstrating how to authenticate and get a build with the Go SDK:
+
+```go
+package main
+
+import (
+    "fmt"
+	"github.com/go-vela/sdk-go/vela"
```
Comment (jbrockopp) [inline content/reference/sdk/go.md]: Super nit-picky but could we fix the spacing here?

File: content/reference/sdk/go.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+)
+
+func main() {
+	// url is the full URI to the Vela server
+	url := "https://your-vela-server.example.com"
+
+    token := "someToken"
+	accessToken := "someAccessToken"
+	refreshToken := "someRefreshToken"
```
Comment (jbrockopp) [inline content/reference/sdk/go.md]: Could we fix the spacing here?

File: content/reference/sdk/go.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    token := "someToken"
+	accessToken := "someAccessToken"
+	refreshToken := "someRefreshToken"   
+
+	// instantiate a new Vela client
+    client, err := vela.NewClient(url, nil)
+    if err != nil {
+        fmt.Println(err)
+    }
```
Comment (jbrockopp) [inline content/reference/sdk/go.md]: spacing here?

The comment and code don't match

File: content/reference/sdk/go.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	// instantiate a new Vela client
+    client, err := vela.NewClient(url, nil)
+    if err != nil {
+        fmt.Println(err)
+    }    
+
+	// set the Authentication mechanism for the client
+	// to use our access token given from Vela
+    client.Authentication.SetTokenAuth(token)
+    client.Authentication.SetAccessAndRefreshAuth(accessToken, refreshToken)
```
Comment (jbrockopp) [inline content/reference/sdk/go.md]: spacing here?

The comment and code don't match

**Review by jbrockopp — CHANGES_REQUESTED** [review]

File: content/reference/sdk/go.md (hunk 1 above)
Comment (kneal) [reply content/reference/sdk/go.md]: Think we got it!

File: content/reference/sdk/go.md (hunk 2 above)
Comment (kneal) [reply content/reference/sdk/go.md]: Think we got it!

File: content/reference/sdk/go.md (hunk 3 above)
Comment (kneal) [reply content/reference/sdk/go.md]: Think we got it!

File: content/reference/sdk/go.md (hunk 4 above)
Comment (kneal) [reply content/reference/sdk/go.md]: Think we got it!

**Review by jbrockopp — APPROVED** [review]
LGTM 🎉

**Review by matt-fevold — APPROVED** [review]

**Pull request merged**